README.md

```diff
@@ -457,6 +462,51 @@ axios.get('/user/12345', {
 })
 ```
 
+## Cancellation
+
+You can cancel a request using a *cancel token*.
+
+> The axios cancel token API is based on [cancelable promises proposal](https://github.com/tc39/proposal-cancelable-promises), which is currently at Stage 1.
+
+You can create a cancel token by passing an executor function to the `CancelToken` constructor as shown below:
+
+```js
+var Cancel = axios.Cancel;
+var CancelToken = axios.CancelToken;
+
+var cancel;
+
+axios.get('/user/12345', {
+  cancelToken: new CancelToken(function executor(c) {
+    // An executor function receives a cancel function as a parameter
+    // You can use the cancel function to cancel the request later
+    cancel = c;
+  })
+}).catch(function(thrown) {
+  if (thrown instanceof Cancel) {
```

Comment:
@rubennorte It makes sense, but my goal was to implement `Cancel` and `CancelToken` as close to the cancelable promises spec (in its current form) as possible. It's tempting to add `isCancellation` (or [`Cancel.isCancel`](https://github.com/tc39/proposal-cancelable-promises/issues/32)), but it can make future migration path to built-in `Cancel` / `CancelToken` harder. Thoughts?